Makefile

```diff
@@ -53,14 +53,18 @@ endif
 # single tests.
 TARGET ?= test
 
+# Pass GIT_SINCE for the range of commits to test. Used with the commitlint
+# target.
+GIT_SINCE := origin/master
+
```

Comment:
Will this causes any issues if we want to run this test on release branch?

Reply:
If you run it locally without passing `GIT_SINCE` on the commandline, you'll get some unexpected output. For PRs the test will need to pass the base branch of the PR (and the CI environment has the information).

Reply:
there might be a good way to detect the parent of the current branch, but I don't know how to do that (and this is sufficient to get started)